pkgs/servers/nosql/riak-cs/stanchion.nix

```diff
@@ -1,11 +1,12 @@
-{ stdenv, lib, fetchurl, unzip, erlang, git, wget, which, pam, coreutils, riak }:
+{ stdenv, lib, fetchurl, unzip, erlang, git, wget, which, pam, coreutils, riak 
+, Carbon ? null, Cocoa ? null }:
 
 stdenv.mkDerivation rec {
   name = "stanchion-2.1.1";
 
   buildInputs = [
     which unzip erlang pam git wget
```

Comment:
I just used it to conform to style, as declared here: https://github.com/NixOS/nixpkgs/blob/580b5435e0ae65d637a26fb205b2b91528116d6c/pkgs/development/interpreters/erlang/R16B02-8-basho.nix